OpenRA.Game/Game.cs

```diff
@@ -36,6 +36,8 @@ public static class Game
 		public static ICursor Cursor;
 		static WorldRenderer worldRenderer;
 
+		public static MouseButtonPreference MouseButtonPreference = new MouseButtonPreference();
```

Comment:
It's a bit ugly to put this on Game.  It would be cleaner to define `ActionButton` and `CancelButton` properties on `Game.Settings.Game`.